core/domain/feedback_services.py

```diff
@@ -237,6 +237,33 @@ def _get_thread_from_model(thread_model):
         thread_model.created_on, thread_model.last_updated)
 
 
+def get_messages_multi_for_exps(exploration_ids, limit=None):
```

Comment:
Do we still need the 'multi' functionality? If not, I recommend removing it, and just restricting to retrieving messages for a single exp id. It's conceptually simpler, and I think this is the general workflow that we want to encourage anyway.